hidtest/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(hidtest C)
+
+set(HIDAPI_HIDTEST_TARGETS)
+if(NOT WIN32 AND NOT APPLE AND CMAKE_SYSTEM_NAME MATCHES "Linux")
+    if(TARGET hidapi_hidraw)
+        add_executable(hidtest_hidraw test.c)
+        target_link_libraries(hidtest_hidraw hidapi_hidraw)
+        list(APPEND HIDAPI_HIDTEST_TARGETS hidtest_hidraw)
```

Comment:
```suggestion
        add_test(NAME hidapi_test_hidraw COMMAND $<TARGET_FILE:hidtest_hidraw>)
```

Reply:
again, this is not a test in terms of CTest